provisioning/service/src/Twin/InitialTwinState.cs

```diff
@@ -11,11 +11,21 @@ namespace Microsoft.Azure.Devices.Provisioning.Service
     /// </summary>
     public sealed class InitialTwinState
     {
+        /// <summary>
+        /// Specifies the initial desired properties on a twin.
+        /// </summary>
+        [JsonIgnore]
+        public InitialTwinPropertyCollection DesiredProperties
+        {
+            get => Properties.Desired;
+            set => Properties.Desired = value;
+        }
+
         /// <summary>
         /// Getter and setter the desired properties.
         /// </summary>
         [JsonPropertyName("properties")]
-        public InitialTwinProperties Properties { get; set; } = new();
+        internal InitialTwinProperties Properties { get; set; } = new();
```

Comment:
We can hide this from the user, but still use it for serialization. They instead access its child directly from `DesiredProperties` above.